content/en/news/2016-04-10-results-of-the-2nd-gd.md

```diff
@@ -0,0 +1,200 @@
+chatlogs][8], was most likely a result of the low participation of
+only 62.5% of the team members, i.e. 5 team members, and one of them
+abstained. It was determined we need to repeat the voting on the
+mailinglist.
+
+Audience, Story, et al.
+-----------------------
+
+Another topic we were unable to decide on was audience and story
+guidelines. It was noted that this discussion cannot be brought to
+conclusion without participation of Bugsbane, who was not available in
+the GD and could not be reached by email, but who is the project’s
+lead artist and thus must have a word to speak on this topic. Still,
+some interesting aspects were noted, which are worth to read [in the
+chatlogs][9]. This was also the topics several documents (including
+story suggestions) were submitted to before. Since there have been no
+decisions, these documents might be of value for further work, and are
+thus referenced here:
+
+<!-- Note to translators: Please do not translate the titles. They are
+copied from the documents themselves -->
+
+* [DNr. 16/1: Unofficial Story Development Document][12]
+* [DNr. 16/2: Story Principles and Decisions for the Secret Chronicles
+  of Dr. M.][13]
+* [DNr. 16/3: The Secret Chronicles of Dr. M. Story Development
+  Document - Version 2][14]
+* [DNr. 16/4: Proposal for a TSC story][15]
+* [DNr. 16/6: Input on Meeting Topics][16]
+
+As a result, we postponed topics 6-10 on the schedule.
+
+Life energy system
+------------------
+
+A more successful [discussion][10] arose around the topic of life
+energy and its involvement into game mechanics. The discussion
+benefitted from thoughts by kirbyfan and Luiji, who missed everything
+prior but managed to join the channel during this topic.
+
+In consent, it was determined that the current one/two hit + lifes +
+saves + gameover system is to be replaced with a Rayman-style system,
+which means that the player is given some amount of Hit points
+(HP). In levels, there will be check points distributed properly (task
+for level designers), and when the player’s HP sink to 0, he is reset
+to the last check point he passed. In this system, there is neither
+manual saving, neither lifes, neither game over. It does not suffer
+from the problem of infinite points (this saves) by repeatedly jumping
+on Army and is logically correctly build, i.e. there is no rivalty
+between different ways to (re)store game progress.
+
+It was further determined that powerups shall not be lost at the first
+hit a player receives, but there have not been further decisions on
+when exactly this shall happen then instead.
+
+Regarding deathly pits, we found that using the Rayman-style progress
+system, we are able to keep death pits, since they will simply have
+the effect of setting the player’s HP to 0, which means resetting him
+to the last check point. The discussion about level designers who need
+to place respawn points for abysses is thus obsolete now.
+
```

Comment:
There was a lot of discussion around not using excessive, tricky jumps over pits in levels going forward.  There seemed to be general agreement on this, but correct me if you disagree.  This sort of offers compromise over the two extremes of lots of tricky, instant deaths and not having instant death at all.

It seems we will have to have a separate discussion and/or vote to decide when power ups are lost, as we forgot to bring this discussion to closure before the end of the meeting.